en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -497,8 +497,9 @@ load fields off of contained associations, you can use ``autoFields()``::
 
     // Select id & title from articles, but all fields off of Users.
     $query->select(['id', 'title'])
-        ->contain(['Users'])
-        ->autoFields(true);
+        ->contain(['Users' => function($q) {
+			return $q->autoFields(true);
+		          }]);
```

Comment:
I'm not sure I understand what you mean. (My knowledge around associations building in the ORM is really limited).

Regardless, the example given is the doc does not do what it says it should do.
Maybe some more general clarification is needed around the ``autoFields()`` method.